src/client/dfuse/dfuse_obj_da.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2017-2021 Intel Corporation.
+ * (C) Copyright 2017-2022 Intel Corporation.
```

Comment:
I thought we weren't updating copyrights in files we don't otherwise modify.

Reply:
the githook that @jolivier23 provided was updated to handle this case on some older versions of git, so you probably are using the old one? but yes let's not update the copyright on files we have not touched.

Reply:
If it wasn't working, we would see hundreds of files here.  This is likely the case where he did a conflicting merge with latest master and there were files that had been changed without copyright changes.  I'm fine with updating them this way in this case.  Otherwise, we probably need another pass to update such copyrights.

Reply:
@mchaarawi are you not using the githook? https://github.com/daos-stack/daos/commit/b5b927105f5809ed36aa8c81a85e475360726c5c

How are we going to catch these missed copyright updates?

Reply:
Yes, I had a conflict with daos.spec and the merge brought in all the copyright changes.

Reply:
not sure why you are asking me considering this is your PR. I have not touched those files that you are modifying here to update the copyright.
i would still not like to see PRs that touch files unrelated to the PR itself, but im not going to hold  this back.

Reply:
The src/client/dfuse/dfuse_obj_da.h file was modified this year in https://github.com/daos-stack/daos/pull/8747

Reply:
oops, yea that was my bad.. not for the all the others though.